backend/src/interface/mensa_parser/model.rs

```diff
@@ -1,88 +1,38 @@
 //! These structs are used for parse operations.
 
-use std::fmt::Display;
-
 use crate::util::{Additive, Allergen, MealType, Price};
 
-/// Canteen-Struct containing all mealplan information of an canteen. Contains raw data.
+/// Canteen struct containing all mealplan information of a canteen. Contains raw data.
 #[derive(Debug)]
 pub struct ParseCanteen {
     /// Name of the canteen.
     pub name: String,
-    /// All related lines.
+    /// All the [lines](ParseLine) situated within the canteen.
     pub lines: Vec<ParseLine>,
 }
 
-impl Display for ParseCanteen {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        let mut lines = String::new();
-        for line in &self.lines {
-            lines.push_str(&format!("{line}\n"));
-        }
-        write!(f, "{}\n{lines}", self.name)
-    }
-}
-
-/// Line-Struct containing all information of an line and their meals. Contains raw data.
+/// Line-Struct containing all information of a line and their meals. Contains raw data.
 #[derive(Debug)]
 pub struct ParseLine {
     /// Name of the line.
     pub name: String,
-    /// All related dishes.
+    /// All [dishes](Dish) served at this [canteen](ParseCanteen) at a particular day.
     pub dishes: Vec<Dish>,
 }
 
-impl Display for ParseLine {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        let mut dishes = String::new();
-        for dish in &self.dishes {
-            dishes.push_str(&format!("{dish}\n"));
-        }
-        write!(f, "{}\n\n{dishes}", self.name)
-    }
-}
-
 /// Dish-Struct containing all information of a meal or side.
 #[derive(Debug)]
 pub struct Dish {
     /// Name of the dish.
     pub name: String,
-    /// Price of the meal for students, employees, guests and pupils.
+    /// Price of the meal for students, employees, guests and pupils. See [Price].
```

Comment:
code links should be made with ``[`Price`]``. Also make sure thaht these links are working. This can best be done by compiling the docs and checking there. (`cargo doc --open`)